vdc/scripts/users/test/test_users.py

```diff
@@ -0,0 +1,110 @@
+import unittest
+from user_data import create_all, create_all_idempotent, delete_all_idempotent
+from google.cloud import storage
+import uuid
+from globals import k8s, gcloud_service
+
+user_id = f'test-user-{uuid.uuid4().hex}'
+google_project = 'hail-vdc'
+kube_namespace = 'default'
+
+
+class TestCreatea(unittest.TestCase):
+    def test_create_all(self):
+        data = create_all(google_project, kube_namespace)
+
+        ksa_name = data['ksa_name']
+        gsa_name = f"projects/-/serviceAccounts/{data['gsa_email']}"
+        bucket_name = data['bucket_name']
+
+        try:
```

Comment:
Should the tests be run in CI? Right now, they are not.

Reply:
I don't know; how do you all treat non-mainline scripts? I'm happy to add it to CI. That feels like the right thing to do?

Reply:
See comment above about what needs to happen for ci tests. See another project for guidance. Would be nice to have a makefile for running some of these scripts as well like the shell script for making secrets.